.idea/modules/compatibility-10/jdk-sa-jdwp.compatibility-10.test.iml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
it is set in the project not to generate iml files, are they really needed here?